### ISSUE TITLE
COLOR output profile anomaly

### ISSUE THREAD

**Issue by RawConvert** [darktable-org/darktable#6103]
Today I edited in 3.3.0 and loaded a profile (XMP) that was created yesterday with 3.2.1.  I run dt in a terminal window, and got this -
paulssd2@shovel2:~$ $HOME/DT-Mast-23Aug20/bin/darktable --cachedir $HOME/.cache/darktable-23aug --configdir $HOME/.config/darktable-23aug
[colorout] could not find requested profile `/home/paulssd2/.config/darktable-10aug/color/out/sRGB_v4_ICC_preference.icc'!
[colorout] could not find requested profile `/home/paulssd2/.config/darktable-10aug/color/out/sRGB_v4_ICC_preference.icc'!

Folder darktable-10aug is the config folder used when I ran 3.2.1.  You can see above I used darktable-23aug today for 3.3.0.
So it looks like dt may have the "absolute" path to the output profile, rather than a "relative" one where it would look for the profile in the actual config folder in use.  I don't know whether this is intended, though the dropdown choice in the output module is presumably looking first at the actual config place, and has no knowledge of the 10aug folder.

A second point is that the message saying can't be found is wrong.  The ICC is in the 10aug folder, in fact it is in both -

paulssd2@shovel2:~$ dir -l /home/paulssd2/.config/**darktable-10aug**/color/out
total 100
-rw-r--r-- 1 paulssd2 paulssd2 25432 Feb  6  2020 RTv2_sRGB.icc
-rw-r--r-- 1 paulssd2 paulssd2   748 Feb  6  2020 RTv4_sRGB.icc
-rwxrwxrwx 1 paulssd2 paulssd2  1384 Jan  5  2018 sRGB-elle-V2-g10.icc
-rwxrwxrwx 1 paulssd2 paulssd2  1232 Jan  5  2018 sRGB-elle-V4-g10.icc
-rw-rw-r-- 1 paulssd2 paulssd2 60960 Jul 28 23:16 **sRGB_v4_ICC_preference.icc**
paulssd2@shovel2:~$ dir -l /home/paulssd2/.config/**darktable-23aug**/color/out
total 100
-rw-r--r-- 1 paulssd2 paulssd2 25432 Feb  6  2020 RTv2_sRGB.icc
-rw-r--r-- 1 paulssd2 paulssd2   748 Feb  6  2020 RTv4_sRGB.icc
-rwxrwxrwx 1 paulssd2 paulssd2  1384 Jan  5  2018 sRGB-elle-V2-g10.icc
-rwxrwxrwx 1 paulssd2 paulssd2  1232 Jan  5  2018 sRGB-elle-V4-g10.icc
-rw-rw-r-- 1 paulssd2 paulssd2 60960 Jul 28 23:16 **sRGB_v4_ICC_preference.icc**
paulssd2@shovel2:~$ 

In summary, there are 2 points (if I am not misunderstanding) -
1. Should profiles be accessed according to the current config, or are "full paths" the correct approach?
2. Currently there appears to be a bug resulting in an erroneous message, not finding the profile, and defaulting to the built-in sRGB (I think that was the result).

**Comment by RawConvert** [comment]
Please ignore the striking through above, the text is correct but not the striking through.  I've no idea where it came from...

**Comment by elstoc** [comment]
> Please ignore the striking through above, the text is correct but not the striking through. I've no idea where it came from

Git uses markdown to format commentsmessages. Any text placed between a pair of double tildes `~~some text~~` is interpreted as being struck through: ~~some text~~.

In order to prevent this you can place any pasted output between backticks. See the 'Inline Code' section of https://guides.github.com/features/mastering-markdown/

**Comment by RawConvert** [comment]
`Thanks, I see!  Actually it was a pair of **single** tildes, that's enough to make it strikeout.`
test 1 singles  ~test~
test 2 doubles ~~test~~
test 3 three-of ~~~test~~~

**Comment by RawConvert** [comment]
Item 2 got discussed here - https://discuss.pixls.us/t/clipping-indicator-all-clipped-in-luminance-only-mode/21744/19 - so I think we can consider it closed.
Is anyone interested in item 1?
